package.json

```diff
@@ -59,6 +59,8 @@
   },
   "scripts": {
     "build": "gulp dist",
+    "dev": "gulp dev",
+    "alias": "gulp dev --useAlias",
```

Comment:
Can we change this script to `dev-alias` or something that is more descriptive than just `alias`?